corehq/apps/app_manager/templates/app_manager/v1/partials/case_list.html

```diff
@@ -116,6 +116,16 @@
 
 {% include 'app_manager/v1/partials/case_list_filtering.html' %}
 
+{{ request|toggle_tag_info:"DETAIL_LIST_TAB_NODESETS" }}
+{% if request|toggle_enabled:'DETAIL_LIST_TAB_NODESETS' %}
+<div data-bind="with: shortScreen">
```

Comment:
Surprised to see this in the case list tab, I'd imagined adding a "Sorting" section to the case detail and putting it there, since it affects long details not short ones. What would you think of moving it?